### PR TITLE
QDMEwald integration

### PR DESCRIPTION
**Pull request by junghans**

### CODE REVIEW

File: src/QDMEwald/CMakeLists.txt [hunk 1]
```diff
@@ -1,196 +1,6 @@
-cmake_minimum_required(VERSION 3.10)
-cmake_policy(VERSION 3.10)
-project(QDMEwald)
```
Comment (junghans) [inline src/QDMEwald/CMakeLists.txt:3]: @JensWehner if you want to keep it a separate project please let me know.

Comment (JensWehner) [reply src/QDMEwald/CMakeLists.txt:3]: what is the difference?

Comment (JensWehner) [reply src/QDMEwald/CMakeLists.txt:3]: I do not care from what I read online

**Comment by junghans** [comment]
Ok, it fails due to warnings, your turn @JensWehner.

**Comment by junghans** [comment]
@rhalver here is one more clang-9 warning:
```
/usr/lib64/ccache/clang++   -I../xtp/src/QDMEwald/src -isystem /usr/include/kokkos -Wall -Wextra -Wpedantic -Wshadow -Wconversion -Wno-sign-conversion -Werror -fopenmp=libomp -O2 -DNDEBUG   -fopenmp -std=c++14 -MD -MT xtp/src/QDMEwald/src/CMakeFiles/dummy.dir/dummy.cpp.o -MF xtp/src/QDMEwald/src/CMakeFiles/dummy.dir/dummy.cpp.o.d -o xtp/src/QDMEwald/src/CMakeFiles/dummy.dir/dummy.cpp.o -c ../xtp/src/QDMEwald/src/dummy.cpp
../xtp/src/QDMEwald/src/dummy.cpp:55:17: error: implicit conversion turns floating-point number into integer: 'double' to 'bool' [-Werror,-Wfloat-conversion]
      q.at(i) = (double)((ix + iy + iz) % 2) ? 1.0 : -1.0;
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~ ~
1 error generated.
```

**Comment by junghans** [comment]
@JensWehner please review.

**Pull request merged**